From: Reinhard Tartler <siretart@tauware.de>
Date: Mon, 28 Jun 2010 22:43:55 +0200
Subject: [PATCH] Tweak doxygen config

exclude some directories we use for packaging from doxygen documentation
---
 Doxyfile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/Doxyfile
+++ b/Doxyfile
@@ -616,7 +616,7 @@ RECURSIVE              = YES
 # excluded from the INPUT source files. This way you can easily exclude a
 # subdirectory from a directory tree whose root is specified with the INPUT tag.
 
-EXCLUDE                =
+EXCLUDE                = debian debian-shared debian-static debian-cmov .pc .git
 
 # The EXCLUDE_SYMLINKS tag can be used select whether or not files or
 # directories that are symbolic links (a Unix filesystem feature) are excluded
